lib/Dialect/RTL/Ops.cpp

```diff
@@ -697,6 +749,19 @@ void MulOp::getCanonicalizationPatterns(OwningRewritePatternList &results,
 
       APInt value, value2;
 
+      // mul(x, c) -> mul( shl(x, log2(c)) ), where c is a power of two
```

Comment:
@cgyurgyik If c is a power of two then mul(x, c) -> shl(x, log2(c)). I think the code is correct but the comment has an extra mul().

Reply:
Thanks!